api/tests/c/indigo-test-issue-269.c

```diff
@@ -0,0 +1,139 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "indigo.h"
+//#include "molecule/structure_checker2.h"
+
+const char* FAILED_MOLECULE_STRING = "The molecule\n"
+                                     "  Ketcher 12222012222D 1   1.00000     0.00000     0\n"
+                                     "\n"
+                                     "  6  6  0     0  0            999 V2000\n"
+                                     "    6.2250   -5.4250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                     "    7.0910   -5.9250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                     "    7.0910   -6.9250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                     "    6.2250   -7.4250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                     "    5.3590   -6.9250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                     "    5.3590   -5.9250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0\n"
+                                     "  1  2  1  0     0  0\n"
+                                     "  2  3  2  0     0  0\n"
+                                     "  3  4  1  0     0  0\n"
+                                     "  4  5  2  0     0  0\n"
+                                     "  5  6  1  0     0  0\n"
+                                     "  6  1  2  0     0  0\n"
+                                     "M  RBC  1   4   2\n"
+                                     "M  END\n";
+
```

Comment:
How about to switch to CPP file and utilize raw string literals (available since C++11)?
Then the test data would look closer to the original file, like:
```
const char* FAILED_MOLECULE_STRING = R"(The molecule
  Ketcher 12222012222D 1   1.00000     0.00000     0

  6  6  0     0  0            999 V2000
    6.2250   -5.4250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    7.0910   -5.9250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    7.0910   -6.9250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    6.2250   -7.4250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    5.3590   -6.9250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
    5.3590   -5.9250    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
  1  2  1  0     0  0
  2  3  2  0     0  0
  3  4  1  0     0  0
  4  5  2  0     0  0
  5  6  1  0     0  0
  6  1  2  0     0  0
M  RBC  1   4   2
M  END
)";

```

Reply:
This test emulates the check() call from C API code for host lang (Java, Python etc.). So its  compilability as C is a subject of testing. I 'm not sure that this minor cosmetics is enough to skip it.

Reply:
OK, agree.